packages/@groupby/elements-cache-driver-plugin/README.md

```diff
@@ -0,0 +1,26 @@
+# GroupBy Elements Cache Plugin
+
+This package contains the GroupBy Elements Cache Driver Plugin class.
+
+## Usage
+
+To use the plugin, simply instantiate it and register it with Core:
+
+```js
+const cacheDriverPlugin = new CacheDriverPlugin();
+core.register(cacheDriverPlugin);
+```
+
+## Events
+
+This plugin listens for and dispatches a number of events. These events are defined in the [`@groupby/elements-events`][elements-events] package.
+
+### Received
+
+* `CACHE_REQUEST`: When received, a request to the cache is made and a response is dispatched with the event name provided in the payload of `CACHE_REQUEST`.
+
+### Dispatched
+
+* A cache response event: Dispatched when a `CACHE_REQUEST` is fulfilled. The event is dispatched under the name provided with the corresponding `CACHE_REQUEST` payload.
```

Comment:
> Is it also worth including an example of what one of these event names could look like?

I can add a recommendation about what it should look like for the `CACHE_REQUEST` event, but we have no idea what it would look like for `CACHE_RESPONSE`. I think more extensive documentation might be better suited elsewhere.